lighthouse-cli/bin.js

```diff
@@ -55,8 +54,6 @@ if (cliFlags.configPath) {
   // Resolve the config file path relative to where cli was called.
   cliFlags.configPath = path.resolve(process.cwd(), cliFlags.configPath);
   config = /** @type {!LH.Config} */ (require(cliFlags.configPath));
-} else if (cliFlags.perf) {
```

Comment:
you're removing --perf ? omg

Reply:
can we add an alias for `--onlyCategories=perf` (is that it?)

Reply:
yeah added back the flag as an alias for `--only-categories=performance` for now, but IMO we should get rid of it for v3 :)